snakeviz/cli.py

```diff
@@ -103,14 +105,20 @@ def main(argv=sys.argv[1:]):
         print('No available port found.')
         return 1
 
+    url = "http://{0}:{1}/snakeviz/{2}".format(hostname, port, filename)
     print(('snakeviz web server started on %s:%d; enter Ctrl-C to exit' %
            (hostname, port)))
+    print(unquote_plus(url))
```

Comment:
On testing this, I realize that we want to print the URL quoted because it's important, especially on Windows, that the browser isn't confused by path components that are part of the URL. So the entire file path should be one big, quoted, isolated component of the URL. It works fine unquoted on my Mac, but I think I remember this quoting business is so that it works on Windows.